cirq/google/engine/engine.py

```diff
@@ -446,7 +511,59 @@ def create_batch_program(
                                             new_program_id,
                                             self.context,
                                             new_program,
-                                            batch_mode=True)
+                                            result_type=ResultType.Batch)
+
+    def create_calibration_program(
+            self,
+            layers: List['cirq.google.CalibrationLayer'],
+            program_id: Optional[str] = None,
+            gate_set: Optional[sgs.SerializableGateSet] = None,
+            description: Optional[str] = None,
+            labels: Optional[Dict[str, str]] = None,
+    ) -> engine_program.EngineProgram:
+        """Wraps a list of calibration layers into an Any for Quantum Engine.
+
+        Args:
+            layers: The calibration routines to execute within a batch.
```

Comment:
There's no mention of a "batch" elsewhere in the calibration documentation; is this still copy pasta or just coincidental terminology conflict? I recommend just removing "within a batch" for clarity.

Reply:
Changed to better explain what is going on.